src/App.js

```diff
@@ -1,11 +1,46 @@
 import "./App.css";
 import React from "react";
 import Form from "./components/Form";
+import ProctectedRoute from "./components/ProctedRoutes";
+import useAuth from "./components/UseAuth";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import Register from "./components/Register";
+import Login from "./components/Login";
 
 function App() {
+  const [isAuth, login, logout] = useAuth(false);
   return (
-    <div>
-      <Form />
+    <div className="container">
+      <h1>Todo List</h1>
+      <Router>
```

Comment:
Usually, we wrap the `<App/>` component with `BrowserRouter`. It should be done as a whole. It can be done in `index.js` or in App.js itself as the top-most parent.